### PR TITLE
Updates from rspec-dev (2020-11-18)

### PR DESCRIPTION
**Pull request by JonRowe**
These are some updates, generated from rspec-dev's rake tasks.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -40,3 +40,30 @@ jobs:
...
+        ruby:
+          - 2.7
+          - 2.6
+          - 2.5
+          - 2.4
+          - 2.3
+          - 2.2
+          - 2.1.9
+      fail-fast: false
+    continue-on-error: ${{ matrix.ruby == '2.7' }}
```
Comment (pirj) [inline .github/workflows/ci.yml:58]: Why?

Comment (JonRowe) [reply .github/workflows/ci.yml:58]: Its out of date, but I need support approved first.

Comment (benoittgt) [reply .github/workflows/ci.yml:58]: I tried to do a similar thing on rspec-rails but I had to move it in step in order to continue-on-error properly https://github.com/rspec/rspec-rails/blob/e1a2ae7beffffa731a293f635320fba786af9189/.github/workflows/ci.yml#L101

**Review by benoittgt — APPROVED** [review]

**Review by pirj — APPROVED** [review]
Please feel free to merge build-related PRs at will.
The only way it could affect anyone else but us is a broken PR build, but they are scarce. And we can fix rather quickly if something goes wrong.

**Comment by JonRowe** [comment]
Builds PRS should only be merged if they improve the builds overall status.

**Comment by JonRowe** [comment]
Shoop shoop, all green GHA

**Comment by benoittgt** [comment]
Yeaaah

**Comment by pirj** [comment]
Insane! 🎉

**Pull request merged**